cloudant-spark-sql/src/main/scala/com/cloudant/spark/common/JsonStoreConfig.scala

```diff
@@ -37,6 +37,8 @@ trait JsonStoreConfig {
   implicit val bulkSize: Int 
   def allowPartition(): Boolean = {true}
   def getOneUrl(): String
+  def getOneUrlExcludeDDoc1(): String = {""}
+  def getOneUrlExcludeDDoc2(): String = {""}
```

Comment:
Let's not initialize these 2 so we can use proper null checking anywhere else in the code.